core/arch/arm/plat-sam/sam_sfr.c

```diff
@@ -25,3 +28,26 @@ vaddr_t sam_sfr_base(void)
 
 	return (vaddr_t)va;
 }
+
+static TEE_Result atmel_sfr_probe(const void *fdt, int node,
+				  const void *compat_data __unused)
+{
+	if (_fdt_get_status(fdt, node) != DT_STATUS_OK_SEC)
+		return TEE_ERROR_GENERIC;
```

Comment:
I guess this will lead an error reported in the log, is that the intent?

Reply:
You are right, this was not intended, I always tested with a secure SFR node. I'll modify that.

Thanks